site/docs/3.5.3/js/vendor/bootstrap.bundle.min.js

```diff
@@ -1,7 +1,7 @@
 /*!
-  * Bootstrap v4.4.1 (https://getbootstrap.com/)
-  * Copyright 2011-2019 The Bootstrap Authors (https://github.com/twbs/bootstrap/graphs/contributors)
-  * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+  * Bootstrap v5.0.2 (https://getbootstrap.com/)
```

Comment:
Do `site/docs/3.5.3/js/vendor/bootstrap.bundle.min.js` and `site/docs/3.5.3/js/vendor/bootstrap.bundle.min.js.map` need to be updated? If not, please let me know."

Reply:
yes, they should be updated